.devcontainer.json

```
@@ -1,3 +1,14 @@
 {
+    "image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:0-18",
```

Why javascript-node?  Also, we can use the images published by `devcontainers/images` here :) 

```suggestion
    "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
```

---

@joshspicer You need node to run the dev container CLI - and we seem to be missing javascript-node there which is a problem actually.